modules/monitoring-specifying-how-a-service-is-monitored-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-for-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored for Cluster Observability Operator
+
+To use the metrics exposed by the sample service you created in the "Creating a `MonitoringStack` object for the Cluster Observability Operator" section, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
+
+You can create this configuration by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. 
+The `ServiceMonitor` object requires a `Service` object. The `PodMonitor` object does not, which enables Prometheus to scrape metrics directly from the metrics endpoint exposed by a pod
```

Comment:
which enables **Prometheus** to scrape metrics directly from the metrics
better change as 
which enables **MonitoringStack** to scrape metrics directly from the metrics

Reply:
Changed